### DISCUSSION TITLE
Should I use next-intl-docs-git-feat-next-13-rsc-next-intl.vercel.app/docs/getting-started or next-intl-docs.vercel.app/docs/getting-started?

### DISCUSSION THREAD

**Discussion by Demianeen** [amannn/next-intl#1245] [Q&A] [answered]
I started to go through the next-intl docs (great docs btw), specifically how to setup next-intl without router path like /en. So, I was a little confused when I found that one version has separate no routing guide while another don't. Which doc should I follow?

**Comment by jd-carroll** [comment, marked as answer]
I think you want to use:
next-intl-docs.vercel.app/docs/getting-started

Not sure where you found that other link, but it appears to be for next-intl v2 (not v3). So I guess if you are using next-intl@v2, then the first link. But generally I think you should upgrade / use the second link.

**Reply by Demianeen** [reply]
Yeah, thanks. It just was the first one I found in some search results
